### ISSUE TITLE
How to early stop the update of branch weights as the paper said?

### ISSUE THREAD

**Issue by JiyueWang** [XingangPan/Switchable-Whitening#8]

**Comment by XingangPan** [comment]
You can split the parameters into 2 groups in the optimizer as below:
![issue1](https://user-images.githubusercontent.com/13579537/79767046-60f2aa00-835b-11ea-87e7-8ebe50ea6b59.png)
![issue2](https://user-images.githubusercontent.com/13579537/79767065-67812180-835b-11ea-96cc-7c19a03388ba.png)
Then you can set the learning rate of SW to zero in the middle of the training.

**Comment by JiyueWang** [comment]
Can you show some results about how serious the overfitting problem is?

**Comment by XingangPan** [comment]
@JiyueWang The initial version of SW is implemented with SVD decomposition, where the lack of early stop would lead to a decrease of about 0.5 scores. The current version of SW uses newton's iteration, which naturally has stochasticity, so the overfitting is alleviated. We observe that the effect of early stop is negligible in this case.
